pkgs/misc/emulators/wine/packages.nix

```diff
@@ -10,13 +10,15 @@ in with src; {
     inherit src version supportFlags;
     pkgArches = [ pkgsi686Linux ];
     geckos = [ gecko32 ];
+    mingwW64Gccs = with pkgsCross; [ mingw32.buildPackages.gcc ];
```

Comment:
mingw32 passed here, so mingwGCC be a better name

Reply:
The naming of the `pkgsCross` MinGW-64 packages is weird and inconsistent. `pkgsCross.mingw32` is actually the 32bit build of MinGW-64 and not the old MinGW project which MinGW-64 was forked from.

The name has to be the same for all builds, so I think it's better to stick to the name of the actual project.

Reply:
Maybe it would make sense to use dashes instead of camel case to make it clearer? Eg. `mingw-w64-gccs`?